packages/core/datadog-configuration.gradle

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+def getConfigurationFilePath() {
+    if (project.hasProperty("datadog")) {
+        if (project.datadog.configurationFilePath) {
+            return project.datadog.configurationFilePath
+        }
+    }
+    return '../../datadog-configuration.json'
+}
+
+task copyDatadogConfigurationFile(type: Copy) {
+    from getConfigurationFilePath()
```

Comment:
you may want to check that file indeed exists before running copy task

Reply:
I agree, currently the task succeeds if the file isn't present. I'll make it fail with an explicit error message.